recipes/sepp/relocate.run-sepp.sh.patch

```diff
@@ -0,0 +1,36 @@
+--- /home/sjanssen/Desktop/sepp/sepp-package/run-sepp.sh	2019-03-19 16:33:17.267182143 +0100
++++ sepp-package/run-sepp.sh	2019-03-19 20:23:08.591954892 +0100
+@@ -111,7 +111,7 @@
+ done
+ 
+ 
+-DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
++DIR=/opt/anaconda1anaconda2anaconda3/share/sepp/
```

Comment:
Are you sure, this change is intended here? To me, this folder looks like something that will only be present during the building of the package, but not in the package itself and thus not on the system where it is eventually installed.

From looking at the original file `run-sepp.sh`, I would think that it will not find the default files that you copied to `$PREFIX/share/sepp/ref/` in `build.sh`, i.e. this would break the following lines once the respective options are called:

https://github.com/smirarab/sepp/blob/060779a0eece74db3367509c6a25d7d65f3e43f1/sepp-package/run-sepp.sh#L114-L137

I think the patch should instead leave the `DIR=` definition as is, as this just finds the location of `run-sepp.sh` itself, and change the resource definitions relative to it, so e.g. change:

https://github.com/smirarab/sepp/blob/060779a0eece74db3367509c6a25d7d65f3e43f1/sepp-package/run-sepp.sh#L119

To:
```
	t="$DIR/../share/sepp/ref/reference-gg-raxml-bl-rooted-relabelled.tre"
```

Reply:
The weird path (`/opt/anaconda1...`) should be found by conda and changed automatically post-install, so that's probably OK.

Reply:
Ah, OK. If conda is that smart, just ignore my comment and feel free to merge if there are no further points.